var/spack/repos/builtin/packages/libusbmuxd/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libusbmuxd(Package):
```

Comment:
Looking at the tarball this seems to be an autotools package, thus I think:
```python
class Libusbmuxd(AutotoolsPackage):
```
might be more appropriate. Or is there a reason for not deriving from that base class?